local-volume/provisioner/pkg/deleter/deleter.go

```diff
@@ -26,6 +26,12 @@ import (
 	"k8s.io/client-go/pkg/api/v1"
 )
 
+const (
+	// kubelet event reason list
+	FailedCleaningLocalPV = "FailedCleaningLocalPV"
```

Comment:
I think we need to add these events to the kubernetes main repo too.  Try looking for where the FailedMount event is defined.  Also, check if there's already a suitable event defined that we can just reuse.